linkcheck/strformat.py

```diff
@@ -95,8 +91,8 @@ def is_encoding (text):
 
 
 def url_unicode_split (url):
-    """Like urlparse.urlsplit(), but always returning unicode parts."""
-    return [unicode_safe(s) for s in urlparse.urlsplit(url)]
+    """Like urllib.parse.urlsplit(), but always returning unicode parts."""
+    return [unicode_safe(s) for s in urllib.parse.urlsplit(url)]
```

Comment:
Out of scope for this PR, but I wonder if this function is still necessary.  I don't think urllib.parse ever deals with byte strings on Python 3.

Reply:
#383 created to track this.